doc-bot.php

```diff
@@ -182,20 +182,20 @@ function plugin( $irc, $data ) {
 				return;
 			}
 
-			$page = file_get_contents( $search . str_replace( ' ', '+', $msg->message ) );
-			preg_match_all( "/plugin-card-top.+?column-name.+?<a.+?href=\"(.+?)\">(.+?)</msi", $page, $matches );
+			$resp = file_get_contents( $search . str_replace( ' ', '+', $msg->message ) );
+			$result = json_decode( $resp );
 
-			if ( ! empty( $matches[1] ) ) {
+			if ( ! isset( $result->plugins ) && count( $result->plugins ) > 0 ) {
```

Comment:
This would cause the lookup to fail, as `! isset( $result->plugins )` contradicts `count( $result->plugins )`

Reply:
This should be resolved, `plugins` should always be available (even if no results), so it serves as a pseudo error check.